frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/storage-tab-validation.ts

```diff
@@ -71,7 +71,10 @@ export const validateStorages = (options: UpdateOptions) => {
 export const setStoragesTabValidity = (options: UpdateOptions) => {
   const { id, dispatch, getState } = options;
   const state = getState();
-  const iStorages = iGetStorages(state, id);
+
+  const iStorages = iGetStorages(state, id).filter((iStorage) =>
+    iGetIn(iStorage, ['disk', 'disk']),
```

Comment:
can this rather be everything except cdroms?

Reply:
In theory it can be LUN or FLOPPY, although it's highly unlikely I think we should still keep this filter to be sure we get only storage objects of type DISK.

Reply:
we are showing all the other devices as well, so we should show the validations for them too IMO. Even if it is more unlikely to have them.

Reply:
Yes I agree, we should show all the validations for all the types.